test/test_wit.py

```diff
@@ -7,37 +7,49 @@ class TestWit(unittest.TestCase):
         pass
 
+    def test_vul_leggerwaarden(self):
+        """test create values of theoretical profiel"""
+        pass
+
     def test_create_leggerpunten(self):
         """test create points of theoretical profiel"""
         
         line = TLine([(0, 0), (0, 10)])
-        TI_waterbr = 10
-        TI_talulbr = 3
-        TI_bodembr = 5
+        line_id = 1
+        name = 'test lijn'
+        ti_waterbr = 10
+        ti_talulbr = 3
+        ti_knkbodr = 8
 
-        profiel_dict = create_leggerpunten(line,TI_waterbr, TI_talulbr, TI_bodembr)
+        profiel_dict = create_leggerpunten(line, line_id, name, ti_waterbr, ti_talulbr, ti_knkbodr)
 
-        self.assertDictEqual(profiel_dict,{'22L': (-5.0, 5.0),'22L_peil': 0.00,
-                    'knik_l': (-2.0, 5.0), 'knik_l_dpt': 0.00,
-                    'knik_r': (3.0, 5.0), 'knik_r_dpt': 0.00,
-                    '22R': (5.0, 5.0), '22R_peil': 0.00})
-        
-        
     def test_update_leggerpunten_diepten(self):
         """test update of profiel_dict with depths of theoretical profiel"""
-        
-        profiel_dict = {'22L': (-5.0, 5.0), '22L_peil': 0.00,
-                    'knik_l': (-2.0, 5.0), 'knik_l_dpt': 0.00,
-                    'knik_r': (3.0, 5.0), 'knik_r_dpt': 0.00,
-                    '22R': (5.0, 5.0), '22R_peil': 0.00}
-        leggerdiepten = (-1.0, -3.5, -3.5, -1.0)
-        
-        profiel_dict = update_leggerpunten_diepten(profiel_dict, leggerdiepten)
+
+        profiel_dict = {'id': 1, 'name': 'test lijn',
+                        'L22': (-5.0, 5.0), 'L22_peil': 0.00,
+                        'knik_l': (-2.0, 5.0), 'knik_l_dpt': 0.00,
+                        'knik_r': (3.0, 5.0), 'knik_r_dpt': 0.00,
+                        'R22': (5.0, 5.0), 'R22_peil': 0.00}
+
+        ti_velden_dict = {'ti_waterp': -1.0,
+                          'ti_diepte': 2.0,
+                          'ti_waterbr': 9.0,
+                          'ti_bodemh': -3.0,
+                          'ti_bodembr': 5.0,
+                          'ti_talulbr': 2.0,
+                          'ti_talurbr': 2.0,
+                          'ti_knkbodr': 7.0}
+
+        profiel_dict = update_leggerpunten_diepten(profiel_dict, ti_velden_dict)
 
         self.assertDictEqual(profiel_dict,
-                             {'22L': (-5.0, 5.0), '22L_peil': -1.00,
-                              'knik_l': (-2.0, 5.0), 'knik_l_dpt': -3.50,
-                              'knik_r': (3.0, 5.0), 'knik_r_dpt': -3.50,
-                              '22R': (5.0, 5.0), '22R_peil': -1.0})
-        
-        
\ No newline at end of file
+                             {'id': 1, 'name': 'test lijn',
+                              'L22': (-5.0, 5.0), 'L22_peil': -1.00,
+                              'knik_l': (-2.0, 5.0), 'knik_l_dpt': -3.0,
+                              'knik_r': (3.0, 5.0), 'knik_r_dpt': -3.0,
+                              'R22': (5.0, 5.0), 'R22_peil': -1.0})
+        # not in output:
```

Comment:
ik heb de test gefixed. De dict hoeft niet gesorteerd te zijn, wel stond er nog een verkeerde key in (met : ervoor) en missen deze 3 eigenschappen. Jij moet maar even kijken of dat inderdaad het gewenste gedrag is.